app/lib/upload_redcap_details.rb

```diff
@@ -77,6 +77,10 @@ def self.question_answer_to_redcap_response(question_answer, destroy, *_)
   def self.user_to_redcap_response(user, *_)
     user_id = user.id
 
+    if UserColumnToRedcapFieldMapping.count == 0
+      return nil
```

Comment:
The `self.perform` function will check for `nil` values and exit before calling `self.call_api`.